backends/p4tools/modules/testgen/core/small_step/table_stepper.cpp

```diff
@@ -189,6 +193,7 @@ const IR::Expression *TableStepper::evalTableConstEntries() {
         // Update all the tracking variables for tables.
         std::vector<Continuation::Command> replacements;
         replacements.emplace_back(new IR::MethodCallStatement(Util::SourceInfo(), tableAction));
+        nextState.set(getGlobalTableHitVar(), IR::BoolLiteral::get(true));
```

Comment:
This code seems to do the same thing as the hit variable. Not clear to me what it does differently?

Reply:
I see. Yes that works because the variable is set with each table call. I'd prefer introducing a table context and then retrieving that context. So a string literal with the table declaration name. It can generalize to more scenarios and the variable has a more specific meaning.